README.md

```diff
@@ -8,6 +8,14 @@ Read-only queries can be sent to the replica, while others have to go to the mai
 The `db-replica` API automatically routes the queries to the correct node.
 It integrates at the `java.sql.Connection` level, so you don't have to hunt down hundreds of queries manually.
 
+## Features
+
+- Automatic switching between main and replica databases (`com.atlassian.db.replica.api.DualConnection`).
```

Comment:
Wanna link to the source file?

Reply:
Yes. But I always forget the syntax.

Reply:
Generally: `[label](address)`
Specifically: ``[`com.atlassian.db.replica.api.DualConnection`](src/main/java/com/atlassian/db/replica/api/DualConnection.java)``